From 222093ae7056e4a708ff1be235a06f58a092f62a Mon Sep 17 00:00:00 2001
From: Ying Zhang <b40530@freescale.com>
Date: Thu, 28 Jan 2016 17:33:20 +0800
Subject: [PATCH 0569/1429] t4/b4: add ceetm device tree nodes

cherry picked from:
e3b77f01ca109fb89ceab99a6a5035f26a0d3d49

Signed-off-by: Haiying Wang <Haiying.Wang@freescale.com>
Reviewed-by: Geoff Thorpe <Geoff.Thorpe@freescale.com>
Reviewed-by: Thomas Trefny <Tom.Trefny@freescale.com>
[Original patch taken from QorIQ-SDK-V2.0-20160527-yocto]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/powerpc/boot/dts/fsl/b4qds.dtsi   |    1 +
 arch/powerpc/boot/dts/fsl/t4240qds.dts |    2 ++
 2 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/arch/powerpc/boot/dts/fsl/b4qds.dtsi b/arch/powerpc/boot/dts/fsl/b4qds.dtsi
index 6d17822..6e23470 100644
--- a/arch/powerpc/boot/dts/fsl/b4qds.dtsi
+++ b/arch/powerpc/boot/dts/fsl/b4qds.dtsi
@@ -218,3 +218,4 @@
 };
 
 /include/ "b4si-post.dtsi"
+/include/ "qoriq-qman-ceetm0.dtsi"
diff --git a/arch/powerpc/boot/dts/fsl/t4240qds.dts b/arch/powerpc/boot/dts/fsl/t4240qds.dts
index 30413be..0d7326a 100644
--- a/arch/powerpc/boot/dts/fsl/t4240qds.dts
+++ b/arch/powerpc/boot/dts/fsl/t4240qds.dts
@@ -752,3 +752,5 @@
 
 /include/ "t4240si-post.dtsi"
 /include/ "qoriq-dpaa-res3.dtsi"
+/include/ "qoriq-qman-ceetm0.dtsi"
+/include/ "qoriq-qman-ceetm1.dtsi"
-- 
1.7.5.4

